include/rcutils/macros.h

```diff
@@ -57,7 +57,7 @@ extern "C"
   #endif
 #else
 // Some other non-Windows, non-cygwin, non-apple OS
-  #define RCUTILS_THREAD_LOCAL _Thread_local
```

Comment:
I think we should add a `#define RCUTILS_THREAD_SUPPORT` flag instead of remote it.

Reply:
Agree, let me modify it

Reply:
Done